src/entry-points/local-file-player/App.svelte

```diff
@@ -65,6 +64,25 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
     videoEl.focus();
   }
 
+  async function onDeleteFile(ind: number) {
+    files = files.filter((file,fileIndex) => fileIndex !== ind)
+    if (currFileInd === ind){
+      if (ind === files.length){
+        // Removed the last file, play the new last file
+        currFileInd = files.length-1
+      }else if (ind === 0){
+        // Removed the first file, play the new first file
+        currFileInd = 0
+      }
+      playFile(currFileInd);
+    }
+    if(ind < currFileInd){
```

Comment:
I think it's better to wrap it inside the `else` part of `if (currFileInd === ind){`, because they can't happen at the same time.